accounts/keys.go

```diff
@@ -0,0 +1,168 @@
+package accounts
+
+import (
+	"crypto/ecdsa"
+	"errors"
+	"io"
+	"os"
+
+	"github.com/howeyc/gopass"
+)
+
+var (
+	errNoKeystoreDir = errors.New("Keystore directory does not exists")
```

Comment:
Usually error descriptions are written starting with lowercase to be able to chain them. For example such description with error formatted as `"failed to do magic: %s"` will emit "failed to do magic: Keystore directory does not exists".